docs/contributors/advanced/sponsor-an-upload.md

```diff
@@ -1,31 +1,97 @@
-(sponsor-an-upload)=
+(how-to-sponsor-an-upload)=
 # How to sponsor an upload
 
-Members of the {term}`Ubuntu Sponsors` and {term}`Ubuntu Security Sponsors`
-teams have the right to sponsor patches or new packages. If you are interested
-in {ref}`sponsoring <sponsorship>`, and have gained upload rights, you can
-apply to join these teams.
-
-See {ref}`path-to-upload-rights` for details on how to apply for upload rights.
+{term}`Ubuntu sponsors` and {term}`Ubuntu security sponsors` have the right to sponsor patches or new packages. If you are interested in {ref}`sponsoring <sponsorship>`, and have gained upload rights, apply to join these teams. See {ref}`path-to-upload-rights` for details on how to apply for upload rights.
 
 
 (sponsor-a-package)=
 ## Sponsor a package
 
-This is similar to uploading your own `.changes` file, except that after
-ensuring that the upload follows all quality standards, you will sign the
-content of the proposing person with your key.
+This is similar to uploading your own `.changes` file, except that after ensuring that the upload follows all quality standards (see {ref}`how-to-review-a-merge-proposal`), you sign the content of the proposing person with your key and upload it to the Ubuntu Archive instead of a PPA.
+
+:::{note}
+These instructions assume the request for sponsorship is in the form of a {term}`merge proposal`. See *How can I convert a debdiff contribution into a git-ubuntu branch?* under {external+git-ubuntu:doc}`howto/patch-pilot` for guidance on working with a 'traditional' contribution in the form of a patch generated by the {manpage}`debdiff(1)` tool.
+:::
+
+1. Clone the package repository and change to the created directory:
```

Comment:
(optional) I think we can probably drop the numbering of these steps - the order is still implied whether they're there or not, so they're not really needed